HW0/P5/P5.py

```diff
@@ -0,0 +1,20 @@
+import matplotlib.pyplot as plt
+import math
+
+counts = range(1, 500)
+
+#For an infinite amount of employees to count a total with N bags,
+#it takes log2(N) seconds.
+infinite_employees_time = [math.ceil(math.log(n, 2)) for n in counts]
+
+#For a single employee, it takes N+1 seconds to count the bags
+single_employees_time = [n+1 for n in counts]
```

Comment:
Would it be n-1 for n in counts? Since if we have 256 bags, we just need 255 seconds to count all the bags.